less/home-page.less

```diff
@@ -47,9 +58,13 @@
     }
   }
 
+button {
+  .button-style(white, 75%, @eastern-blue, 10px )
+}
+
   .content-pick {
     padding-top: 30px;
-    border-top: 2px dashed @navigation-border;
+    //border-top: 2px dashed @navigation-border;//
     display: flex;
     justify-content: space-between;
 
```

Comment:
Utilize your @mobile variable in your media queries to make mobile version of page.